.github/workflows/trigger-tests-on-packages.yml

```diff
@@ -0,0 +1,17 @@
+name: Trigger tests for APT packages
+
+on:
+  push:
```

Comment:
Let's discuss when and how should these checks run.

I believe `stable` should be started by a release tag trigger, for example:
https://github.com/syslog-ng/syslog-ng/blob/master/.github/workflows/publish-docker-image.yml#L7

But the best would be if we could also test the production docker image itself (before publishing it).

`nightly` can be scheduled, just as you mentioned.

Reply:
Thanks for the review comment. I have fixed the triggering time on new jobs. Let see I they runs correctly.
I am agreed to test our production image. I will think about it,